src/System.Private.CoreLib/shared/Interop/Windows/BCrypt/Interop.BCryptGenRandom.cs

```diff
@@ -10,24 +10,26 @@ internal partial class Interop
 {
     internal partial class BCrypt
     {
-        internal const int BCRYPT_USE_SYSTEM_PREFERRED_RNG = 0x00000002;
+        internal static unsafe NTSTATUS BCryptGenRandom(byte* pbBuffer, int count)
+        {
+            Debug.Assert(pbBuffer != null);
+            Debug.Assert(count >= 0);
+
+            return BCryptGenRandom(IntPtr.Zero, pbBuffer, count, BCRYPT_USE_SYSTEM_PREFERRED_RNG);
+        }
 
-        internal const uint STATUS_SUCCESS = 0x0;
-        internal const uint STATUS_NO_MEMORY = 0xc0000017;
+        private const int BCRYPT_USE_SYSTEM_PREFERRED_RNG = 0x00000002;
 
         [DllImport(Libraries.BCrypt, CharSet = CharSet.Unicode)]
-        internal static unsafe extern uint BCryptGenRandom(IntPtr hAlgorithm, byte* pbBuffer, int cbBuffer, int dwFlags);
+        private static unsafe extern NTSTATUS BCryptGenRandom(IntPtr hAlgorithm, byte* pbBuffer, int cbBuffer, int dwFlags);
     }
 
     internal static unsafe void GetRandomBytes(byte* buffer, int length)
     {
```

Comment:
This method should not be necessary because of it moved to Path.Win32.cs. We should either keep this or Path.Win32.cs.